Resources/public/js/recommendationsupport.js

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+(function (global, doc) {
+    var eZ = global.eZ = global.eZ || {};
+
+    /**
+     * Contains logic needed to fetch and display YooChoose recommendations
```

Comment:
The code organization with 2 different components, one for fetching, one for displaying was much better, any reason to have merged them into one ?